requirements.txt

```diff
@@ -0,0 +1,2 @@
+nose==1.3.7
```

Comment:
We can now remove `requirements.txt` since this is not a dependency on voluptuous but a dependency on tests.